recipe/meta.yaml

```diff
@@ -10,9 +10,8 @@ source:
   sha256: 5570b451eff4d11962b4f6155f5157eda5427c9db0f8193c644043ce3fc6af73
 
 build:
-  skip: true  # [py<38 or osx]
```

Comment:
macOS builds are being added and conda-forge now builds for Python 3.9 to Python 3.13.

Reply:
Ah, I guess the reason for the original skip of macOS was build error related, given @chrisburr suggested this in https://github.com/conda-forge/staged-recipes/pull/26591. Though without digging into it I'm not clear why given https://github.com/scikit-hep/pyhepmc has passing macOS CI and https://pypi.org/project/pyhepmc/2.13.4/#files has macOS wheels.